Assets/Scripts/SS3D/Systems/Clothing/ClothingSlot.cs

```diff
@@ -0,0 +1,50 @@
+using Coimbra;
+using SS3D.Systems.Inventory;
+using UnityEngine;
+
+namespace SS3D.Systems.Clothing
+{
+    public class ClothingSlot : InventorySlot
```

Comment:
If you use inheritance here, there should probably be methods on the parent class for adding and removing items from the slot. Otherwise why have a base class at all?

Reply:
We can ignore these for now, that will be added later on.